tests/components/gree/test_climate.py

```diff
@@ -344,8 +344,32 @@ async def test_send_command_device_timeout(hass, discovery, device, mock_now):
     assert state.state != STATE_UNAVAILABLE
 
 
-async def test_send_power_on(hass, discovery, device, mock_now):
-    """Test for sending power on command to the device."""
+async def test_send_power_on_device_timeout(hass, discovery, device):
```

Comment:
Why are we changing the climate tests in this PR that should be about adding more switches?

Reply:
It add's a bit more coverage, but I can also remove it if you prefer.

Reply:
Yes, please make a separate PR with the climate test changes.